6.bulk_Morphology_Elastic_Network/README.md

```diff
@@ -0,0 +1,5 @@
+# Morphological features Predict Secretome Protein Abundance via Elastic Net
+
+Here we show that morphological features can be used to predict the abundance of secretome proteins in a cell. We use the elastic net algorithm to predict the abundance of secretome proteins in a cell.
```

Comment:
Consider adding links to definitions for `secretome proteins` and `elastic net algorithm` to help define these items in the context of the readme.

Reply:
The nomic secretome proteins aren't defined anywhere except the data...I will raise this as an issue and create a README to address this.